app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/RetrofitClientInstance.kt

```diff
@@ -1,18 +1,18 @@
-package edu.uc.kansakpb.ikeahjalpare3.ui.main
+package edu.uc.kansakpb.ikeahjalpare3
 
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://ikeafurnituredb.s3.us-east-2.amazonaws.com"
+    private var retrofit: Retrofit? = null
+    private const val BASE_URL = "https://ikeafurnituredb.s3.us-east-2.amazonaws.com"
```

Comment:
Remove unnecessary semi-colon. Define `BASE_URL` as `const` as the value will never change.